beacon_node/lighthouse_network/src/rpc/methods.rs

```diff
@@ -24,6 +24,13 @@ use types::{
 pub type MaxErrorLen = U256;
 pub const MAX_ERROR_LEN: u64 = 256;
 
+/// The max number of blobs we expect in the configs to set for compile time params.
+/// Note: This value is an estimate that we should use only for rate limiting,
+/// bounds checking and other non-consensus critical operations.
+///
+/// For exact value, we should always check the chainspec.
+pub const MAX_BLOBS_PER_BLOCK_CEILING: u64 = 16;
```

Comment:
I think we could even go with a higher number if it's for non critical checks and preventing DOS, so we don't run into issues in upcoming upgrades and we don't have to frequently update this?

Reply:
I thought about this for https://github.com/sigp/lighthouse/pull/6462, but as far as I understood this value may change per fork right?

Reply:
Yeah it can change, but like i mentioned in the comment, it is to be used only for rate limiting and bounds checking of the incoming rpc request, so we can bump the number up in preparation for a fork